src/sentry/replays/endpoints/project_replay_details.py

```diff
@@ -75,4 +129,4 @@ def delete(self, request: Request, project: Project, replay_id: str) -> Response
             return Response(status=404)
 
         delete_recording_segments.delay(project_id=project.id, replay_id=replay_id)
-        return Response(status=202)
+        return Response(status=204)
```

Comment:
good catch!!

Reply:
Just need to update the test coverage to assert 204 instead of 202.